History.md

```diff
@@ -198,7 +198,7 @@ N/A
   [Issue #10042](https://github.com/meteor/meteor/issues/10042)
 
 * Mongo has been updated to version 4.0.2 and the `mongodb` npm package
-  used by `npm-mongo` has been updated to version 3.1.1.
+  used by `npm-mongo` has been updated to version 3.1.6.
```

Comment:
I believe this will cause deprecation warnings because [`Collection#insert` is deprecated since 3.1.2](https://github.com/mongodb/node-mongodb-native/commit/a5d0f1d7e1fc3c39ac81fa31e04f2433f86ac183#diff-a3644fc6e558da2bde71112d24ec75baR664).

Reply:
A lot of other `Collection` methods, too. Can we disable the deprecation warnings somehow?

Reply:
The driver uses Node's `util.deprecate` function. As far as I'm aware, we can only disable *all* deprecation warnings (with `process.noDeprecation = true`) but not just those for `mongodb`.

Reply:
Fortunately, the `process.noDeprecation` check only happens once, when the function is first wrapped by `util.deprecate`, so we can temporarily set `process.noDeprecation = true` while importing the `mongodb` package: 2a8dea89349935ff0e0452e5d1a1f5b09df954ad